release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -466,19 +466,115 @@ With this update, the default credentials request for AWS has been modified to a
+
+This features supports the following storage types:
+
+* Amazon Web Services (AWS) Elastic Block storage (EBS) (for more information, see ???)
+
+* Microsoft Azure Disk storage (for more information, see ???)
+
+* Google Cloud Platform (GCP) persistent disk (PD) storage (for more information, see ???)
+
+[id="ocp-4-13-storage-csi-vol-detach-non-graceful-shutdown"]
+==== Detach CSI volumes after non-graceful node shutdown (Technology Preview)
+Container Storage Interface (CSI) drivers can now automatically detach volumes when a node goes down non-gracefully. When a non-graceful node shutdown occurs, you can then manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc[Detach CSI volumes after non-graceful node shutdown].
+
+[id="ocp-4-13-storage-vsphere-encryption-support"]
+==== VMware vSphere encryption support is generally available
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on vSphere. 
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#vsphere-pv-encryption[vSphere persistent disks encryption].
+
+[id="ocp-4-13-storage-vpshere-top-support-multi-datacenters"]
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters and to set up the topology using failure domains created during installation or post-installation.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage class defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class. While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
```

Comment:
Correct, having two default in the long run is still not recommended. Also we fire an alert after 10 minutes: https://github.com/openshift/cluster-storage-operator/blob/master/manifests/12_prometheusrules.yaml#L15-L27

Maybe we could just mention that users will get an alert if the don't ensure only one default is present?